### PR TITLE
Unify crowbar and SUSE design

### PR DESCRIPTION
**Pull request by rsalevsky**
@aplanas Finally the last one. ;)

### CODE REVIEW

**Comment by aplanas** [comment]
LGTM: +1

**Comment by vuntz** [comment]
Can you post a screenshot?

**Comment by rsalevsky** [comment]
crowbar index

**Comment by vuntz** [comment]
@rsalevsky isn't the green in menu something that should be SUSE-specific?

**Comment by rsalevsky** [comment]
Somehow yes but the blue was dell specific. So changing from blue to green has no downside in general.

**Comment by vuntz** [comment]
> Somehow yes but the blue was dell specific. So changing from blue to green has no downside in general.

Except that blue is being used for links (see the node link) and, well, it's kind of the standard on the web. It's weird to have green and blue, and I'd definitely argue that green is suse-specific.

**Comment by rsalevsky** [comment]
crowbar

File: crowbar_framework/app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -21,19 +21,22 @@
...
+    %footer
       = render :partial => "shared/footer"
 
-    %script#message-template{ :type => "text/x-handlebars-template" }
-      .alert{ :class => "{{classes}}" }
-        {{#if dismiss}}
-        %button.close{ :type => "button", "data-dismiss" => "alert", "aria-hidden" => true }
-          &times;
-        {{/if}}
+      %script#message-template{ :type => "text/x-handlebars-template" }
```
Comment (tboerger) [inline crowbar_framework/app/views/layouts/application.html.haml]: Wrong indention

File: crowbar_framework/app/assets/stylesheets/content/footer.css.scss [hunk 2]
```diff
@@ -1,6 +1,8 @@
 footer {
+  color: #999;
+  font-size: 12px;
+  margin-top: 40px;
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/content/footer.css.scss]: IMHO the padding before had been totally fine. Change this only if you really add a sticky footer.

File: crowbar_framework/app/assets/stylesheets/variables.css.scss [hunk 3]
```diff
@@ -6,6 +6,10 @@ $gray: lighten(#000, 33.5%);
 $gray-light: lighten(#000, 60%);
 $gray-lighter: lighten(#000, 93.5%);
 
+// Navbar colors
+// -------------------------
+$navbar-font-color: #bebebe;
```
Comment (tboerger) [inline crowbar_framework/app/assets/stylesheets/variables.css.scss]: I think the name of this is pretty wrong as it's note related only to navbars, it's related more general to dropdowns as well.

**Comment by tboerger** [comment]
Gating fail is expected.
:+1: Merging.

**Pull request merged**